addons/udes_warehouse_classification/models/stock_picking.py

```diff
@@ -6,16 +6,24 @@ class StockPicking(models.Model):
     _inherit = "stock.picking"
 
     def _get_classification_messages_for_product_picking(self):
-        """Return the product with classification messages appropriate for a picking"""
+        """
+        Return all products with classification messages appropriate to warn users. It is getting
+        all products, not only the products of the picking because it may be some products with
+        classifications that are not inside the picking but we still need to warn users.
```

Comment:
```suggestion
        Return classification messages for all products that have one.
        Note: returns all products because we may still need to warn 
        users if scan a product that was not expected on a picking
```

If you like my suggestion, use it, if not, feel free to resolve it.

Reply:
I found it not confusing , not even a little. Maybe because i know what i wrote :)
Do you have any suggestion how to write, as i am afraid that you still will not like.

Reply:
That makes sense :D I wrote a suggestion there with the docstring paraphrased, can you not see it? Let me know